aws-android-sdk-s3/src/main/java/com/amazonaws/services/s3/S3ClientOptions.java

```diff
@@ -305,6 +352,22 @@ public boolean isPayloadSigningEnabled() {
         return payloadSigningEnabled;
     }
 
+    /**
+     * <p>
+     * Returns whether the client skips content MD5 check for all requests
+     * and responses.
+     * </p>
+     * <p>
+     * Setting this flag will allow the client to disregard data transfer
+     * integrity check by bypassing content MD5 check for all requests and
+     * responses.
+     * </p>
+     *
+     * @param skipContentMd5Check True to always skip content MD5 check
+     */
+    public void setSkipContentMd5Check(boolean skipContentMd5Check) {
```

Comment:
Suggestion to rename to: `skipContentMd5Check`

Reply:
this overlaps with the flag's variable name unfortunately

Reply:
didn't matter. will change.